src/core/AudioEngine.cpp

```diff
@@ -67,7 +67,6 @@ namespace lmms
 using LocklessListElement = LocklessList<PlayHandle*>::Element;
 
 static thread_local bool s_renderingThread;
-static thread_local bool s_runningChange;
```

Comment:
Looks like this may have been read before being initialized.

You may want to initialize `s_renderingThread` to false just in case.

Reply:
From cppreference:

```
Static initialization

There are two forms of static initialization:
1) If possible, [constant initialization](https://en.cppreference.com/w/cpp/language/constant_initialization) is applied.
2) Otherwise, non-local static and thread-local variables are [zero-initialized](https://en.cppreference.com/w/cpp/language/zero_initialization).

In practice:

Constant initialization is usually applied at compile time. Pre-calculated object representations are stored as part of the program image. If the compiler doesn't do that, it must still guarantee that the initialization happens before any dynamic 

Variables to be zero-initialized are placed in the .bss segment of the program image, which occupies no space on disk and is zeroed out by the OS when loading the program.
```

I can set it equal to `false` to do constant initialization to be safe, but I'm pretty sure these were zero initialized, which should have the same effect.

Reply:
However, I could be wrong, and maybe the rules for default initialization apply. Initialization for this language is never clear to me.